src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -2298,6 +2300,29 @@ public SyntaxNode LambdaParameter(string identifier, ITypeSymbol type)
         /// </summary>
         public abstract SyntaxNode TupleExpression(IEnumerable<SyntaxNode> arguments);
 
+        /// <summary>
+        /// Parses an expression from string
+        /// </summary>
+        internal abstract SyntaxNode ParseExpression(string stringToParse);
+
+        internal abstract SyntaxToken CommaTokenWithElasticSpace();
+
+        internal abstract SyntaxTrivia Trivia(SyntaxNode node);
+
+        internal abstract SyntaxNode DocumentationCommentTrivia(IEnumerable<SyntaxNode> nodes, SyntaxTriviaList trailingTrivia, SyntaxTrivia lastWhitespaceTrivia, string endOfLineString);
+
+        internal abstract bool IsNamedArgument(SyntaxNode syntaxNode);
+
+        internal abstract bool IsWhitespaceTrivia(SyntaxTrivia trivia);
```

Comment:
already on ISyntaxFacts? (or should be there).

Reply:
Already removed from here in the next PR.